src/frameworks/docs/nuxt.md

```diff
@@ -0,0 +1,64 @@
+# Integrate Nuxt
+
+Using the Firebase CLI, you can deploy your Nuxt apps to Firebase and
+serve them with Firebase Hosting. The CLI respects your Nuxt settings and
+translates them to Firebase settings with zero or minimal extra configuration on
+your part. If your app includes dynamic server-side logic, the CLI deploys that
+logic to Cloud Functions for Firebase.
+
+Note: Framework-aware Hosting is an early public preview. This means
+that the functionality might change in backward-incompatible ways. A preview
+release is not subject to any SLA or deprecation policy and may receive limited
+or no support.
+
+## Before you begin
+
+Before you get started deploying your app to Firebase,
+review the following requirements and options:
+
+- Firebase CLI version 12.1.0 or later. Make sure to
+  [install the CLI](https://github.com/firebase/firebase-tools#installation)
+  using your preferred method.
+- Optional: Billing enabled on your Firebase project
+  (required if you plan to use SSR)
```

Comment:
If an existing Nuxt project is required, would that mean that "Initialize a new project" as detailed below doesn't work?

Reply:
Hmm let me refactor this.

Reply:
This seems to refer to two separate "projects": one being the firebase hosting project, and the other is referring to your nuxt project.

Reply:
@egilmorez thanks for pointing that out! @JulienMartel I was checking and actually the Nuxt implementation does not have an init method, therefore Nuxt isn't an option in `firebase init hosting`.

We can remove `Initialize a new project` in this PR and add it back after adding the `init` method to Nuxt, which should be simple.

Reply:
Sounds good. Thanks for catching that.